src/codegen/ast_interpreter.cpp

```diff
@@ -285,6 +285,11 @@ class RegisterHelper {
 Value ASTInterpreter::execute(ASTInterpreter& interpreter, CFGBlock* start_block, AST_stmt* start_at) {
     threading::allowGLReadPreemption();
 
+#if SLOWPATH_TIMERS
+    static int _stid = Stats::getStatId("zzz_slowpath_astinterpreter_execute_us");
+    StatTimer _st(_stid);
+#endif
```

Comment:
yeah, that was already on my todo list :)

On Wed, Apr 29, 2015 at 2:20 PM, Kevin Modzelewski <notifications@github.com
> wrote:

> In src/codegen/ast_interpreter.cpp
> <https://github.com/dropbox/pyston/pull/475#discussion_r29382242>:
>
> > @@ -285,6 +285,11 @@ class RegisterHelper {
> >  Value ASTInterpreter::execute(ASTInterpreter& interpreter, CFGBlock* start_block, AST_stmt* start_at) {
> >      threading::allowGLReadPreemption();
> >
> > +#if SLOWPATH_TIMERS
> > +    static int _stid = Stats::getStatId("zzz_slowpath_astinterpreter_execute_us");
> > +    StatTimer _st(_stid);
> > +#endif
>
> I would make this into a macro :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dropbox/pyston/pull/475/files#r29382242>.
>